src/appmixer/google/drive/NewFile/NewFile.js

```diff
@@ -1,13 +1,13 @@
-'use strict';
-const Promise = require('bluebird');
 const moment = require('moment');
 const { google } = require('googleapis');
 const commons = require('../drive-commons');
-const { URL, URLSearchParams } = require('url');
 
 const getNewFiles = async (lock, drive, folder, pageToken, newFiles = []) => {
 
-    const { data: { changes, newStartPageToken, nextPageToken } } = await drive.changes.list({ pageToken, fields: '*', includeRemoved: false, includeDeleted: false });
+    const { data: { changes, newStartPageToken, nextPageToken } } = await drive.changes.list({
```

Comment:
Missing new version in component.json

Reply:
And missing new `patch` version in bundle.json and new `minor` version in module.json (because drive-commons.js has a new function)

Reply:
@vtalas I still cannot see a version increased in the bundle.json file.